clap_generate/src/lib.rs

```diff
@@ -181,10 +181,8 @@ where
 {
     app.set_bin_name(bin_name);
 
-    if !app.is_set(clap::AppSettings::Built) {
-        app._build();
-        app._build_bin_names();
-    }
+    app._build();
+    app._build_bin_names();
```

Comment:
This function will be repeated since it doesn't have a check inside.

Reply:
Done.